snapcraft/internal/lxd.py

```diff
@@ -144,18 +147,21 @@ def _get_container_status(self):
     def execute(self, step='snap', args=None):
         with self._ensure_started():
             self._setup_project()
-            self._wait_for_network()
-            self._container_run(['apt-get', 'update'])
-            self._inject_snapcraft()
-            command = ['snapcraft', step]
-            if step == 'snap':
-                command += ['--output', self._snap_output]
-            if self._host_arch != self._project_options.deb_arch:
-                command += ['--target-arch', self._project_options.deb_arch]
-            if args:
-                command += args
             try:
-                self._container_run(command, cwd=self._project_folder)
+                if step == 'refresh':
```

Comment:
Again, not a lifecycle step. Please change terminology or refactor this function a bit.